config/locales/en.yml

```diff
@@ -139,6 +139,9 @@ en:
     "h":
       name: "Qualification round"
       cell_name: "Qualification round"
+    "h2h":
+      name: "Head-to-Head Final"
+      cell_name: "Head-to-Head Final"
```

Comment:
Are we sure that (even in the long run, when we have "proper" support) finals are the only rounds that can appear as H2H?

And even if that's the case from a *current 2026 Regulations* perspective: Wouldn't it make more sense to define H2H as a separate "add-on" string, and glue it in front of whatever the round type originally was?